tests/unittests/parsing/VisitorTests.cpp

```diff
@@ -694,6 +694,7 @@ class C; endclass
     CHECK(SyntaxPrinter::printFile(*newTree) == R"(
 `default_nettype none
 `unconnected_drive pull0
+`celldefine
```

Comment:
You added this to the *expected* output string. You should also add it up above to the actual input, and then this test should pass.